lib/Dialect/CGGI/Transforms/Passes.td

```diff
@@ -19,4 +19,20 @@ def SetDefaultParameters : Pass<"cggi-set-default-parameters"> {
   let dependentDialects = ["mlir::heir::cggi::CGGIDialect"];
 }
 
+def BooleanLineVectorizer : Pass<"cggi-boolean-line-vectorize"> {
```

Comment:
nit: instead of "line vectorizer" it should be "straight line vectorizer". You could also just call it `BooleanVectorizer`